spec/cli_spec.rb

```diff
@@ -6,7 +6,10 @@
 
 RSpec.describe 'CLI' do
   it 'renders reproducible dumps' do
-    system('bundle', 'exec', 'rails-response-dumper', chdir: APP_DIR, exception: true)
+    system('bundle', 'exec', 'rails-response-dumper', chdir: APP_DIR, exception: false,
+                                                      err: %w[tmp/test_error_log w])
```

Comment:
Please format one argument per line and put the closing `)` on its own line.

Making this `exception: true` means we won't catch buggy dumps. Maybe we should move this "after hook" test to its own `it` or spec.